packages/calendar/src/util/date/Translations.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  cs,
+  da,
+  de,
+  enGB,
+  et,
+  es,
+  fi,
+  fr,
+  it,
+  lt,
+  lv,
+  nl,
+  nb,
+  pl,
+  ru,
+  sv
+} from "date-fns/locale";
+
+const locales = {
+  cs,
+  da,
+  de,
+  enGB,
+  et,
+  es,
+  fi,
+  fr,
+  it,
+  lt,
+  lv,
+  nl,
+  nb,
+  pl,
+  ru,
+  sv
+};
+const weekName = {
+  cs: "T",
+  da: "u",
+  de: "w",
+  enGB: "w",
+  et: "n",
+  es: "s",
+  fi: "v",
+  fr: "s",
+  it: "s",
+  lt: "s",
+  lv: "s",
+  nl: "w",
+  nb: "u",
+  pr: "t",
+  ru: "н",
+  sv: "v"
+};
+
+export type Language =
+  | "cs"
+  | "da"
+  | "de"
+  | "enGB"
+  | "et"
+  | "es"
+  | "fi"
+  | "fr"
+  | "it"
+  | "lt"
+  | "lv"
+  | "nl"
+  | "nb"
+  | "pl"
+  | "ru"
+  | "sv";
+
+export interface Translations {
+  locale: Locale;
+  weekName: string;
+}
+
+export const getTranslations = (language: Language = "enGB"): Translations => {
+  return {
+    locale: locales[language],
```

Comment:
Maybe the user could be required to pass the language object into the getTranslationsFn? 

- It would move the languages logic outside of webui (you can add / remove languages more easily). 
- And also keep unused languages from increasing the bundle size.